blint/cli.py

```diff
@@ -238,6 +238,8 @@ def handle_args():
     args = build_args()
     if not args.no_banner and args.subcommand_name != "sbom":
         print(BLINT_LOGO)
+    if not args.src_dir_image:
```

Comment:
This will also apply when `args.use_sbom` is `False`. Is the `os.getcwd()` default also intended to apply in that case? The argument description for `-i` indicates that the current directory is the default in both cases, but the default value is missing for the non-SBOM case (https://github.com/owasp-dep-scan/blint/blob/main/blint/cli.py#L50-L53). I can update the default to only apply in SBOM mode if desired.